src/graph/utils/sycl_check.hpp

```diff
@@ -20,24 +20,12 @@
 #ifdef DNNL_WITH_SYCL
 #if __has_include(<sycl/sycl.hpp>)
 #include <sycl/sycl.hpp>
-#elif __has_include(<CL/sycl.hpp>)
-#include <CL/sycl.hpp>
 #else
 #error "Unsupported compiler"
 #endif
 
-#if defined(__INTEL_LLVM_COMPILER)
-#if (__INTEL_LLVM_COMPILER < 20230000)
-#define DNNL_USE_SYCL121_API 1
-#else
-#define DNNL_USE_SYCL121_API 0
-#endif
-#elif defined(__LIBSYCL_MAJOR_VERSION)
-#if (__LIBSYCL_MAJOR_VERSION < 6)
-#define DNNL_USE_SYCL121_API 1
-#else
-#define DNNL_USE_SYCL121_API 0
-#endif
+#if defined(__INTEL_LLVM_COMPILER) && __INTEL_LLVM_COMPILER >= 20230000
```

Comment:
Why do we keep this code?

Reply:
Theoretically now that I removed the `CL/sycl.hpp` headers `#if __has_include(<sycl/sycl.hpp>)` implied high enough SyCL version, so this check can be removed as redundant.